aten/tools/run_tests.sh

```diff
@@ -27,48 +27,29 @@ python test/run_test.py --cpp --verbose -i \
   cpp/legacy_vmap_test \
   cpp/operators_test
 
-if [[ -x ${CPP_TESTS_DIR}/tensor_interop_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/tensor_interop_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cudnn_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cudnn_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_generator_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_generator_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/apply_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/apply_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/stream_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/stream_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_half_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_half_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_vectorized_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_vectorized_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_distributions_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_distributions_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_optional_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_optional_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_tensor_interop_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_tensor_interop_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_complex_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_complex_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_complex_math_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_complex_math_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_cub_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_cub_test
-fi
-if [[ -x ${CPP_TESTS_DIR}/cuda_atomic_ops_test ]]; then
-  python test/run_test.py --cpp --verbose -i cpp/cuda_atomic_ops_test
-fi
+run_if_exists() {
+  local test_name="$1"
+  if [ -f "${CPP_TESTS_DIR}/${test_name}" ]; then
```

Comment:
`[[` as before? (not sure about the difference in this particular case, but I've read that "[[ has fewer surprises and is generally safer to use".

Reply:
I wonder if one of our linters should have picket it up per https://google.github.io/styleguide/shellguide.html#test----and---